warp-tls/Network/Wai/Handler/WarpTLS.hs

```diff
@@ -81,6 +83,24 @@ data TLSSettings = TLSSettings {
     -- Default: '[TLSExtra.cipher_AES128_SHA1, TLSExtra.cipher_AES256_SHA1, TLSEtra.cipher_RC4_128_MD5, TLSExtra.cipher_RC4_128_SHA1]'
     --
     -- Since 1.4.2
+  , tlsWantClientCert :: Bool
+    -- ^ Whether or not to demand a certificate from the client.  If this
+    -- is set to True, you must handle received certificates in a server hook
+    -- or all connections will fail.
+    --
+    -- Default: False
+  , tlsServerHooks :: TLS.ServerHooks
+    -- ^ Whether or not to demand a certificate from the client.  If this
+    -- is set to True, you must handle received certificates in a server hook
+    -- or all connections will fail.
+    --
+    -- Default: False
```

Comment:
Is the `tlsWantClientCert` documentation supposed to be == to the `tlsServerHooks` documentation?